dev-docs/source/auth.rst

```diff
@@ -71,18 +78,28 @@ The file format provides various options, such as:
         }
     }
 
-A user may have multiple profiles.
-Each profile gives the user additional rights to view a particular resource.
-
-The ``mandatoryFilterSets`` setting ensures that a listing can only be requested
-when a certain set of filters is are given. For example, a frontend office employee
+This scope is called ``medewerker``.
+It applies to all requests that carry the scope ``BRP/R``.
+If more than one scope is listed, all scopes must be carried for the profile to apply.
+By implication, an empty ``scopes`` denotes a profile that always applies.
+
+The ``datasets`` part of the profile lists permissions granted beyond those
+that are granted to the scope(s) by the schema.
+Permissions already granted by the schema are never taken away.
+The permissions granted may be restricted to requests that query particular fields.
+With the example profile, requests with scope ``BRP/R``
+gain permission to read the field ``bsn`` on the table ``ingeschrevenpersonen``,
+provided that the request queries for either ``bsn`` and ``lastname``,
+or ``postcode`` and ``lastname`` (or all three fields).
+
+The intention behind ``mandatoryFilterSets`` is to ensure
+that listings are restricted on a need-to-know basis.
+For example, a profile might express that a frontend office employee
 may only access data of someone when they can provide their last name and postal code.
-
-Another case for profiles would be a statistician, that may only read the age and neighbourhood
+Or a statistician might be allowed to read age and neighbourhood
 fields to aggregate data, without ever having access to identifiable data.
-Encoding such feature into the schema file amongst other rules would produce
-very complex ``auth`` fields, while the profile file gives a
-fitting definition of such role and permissions.
+Encoding such rules in the schema file using a custom scope
+would require every other request that accesses the dataset to also have this scope.
```

Comment:
May add some explanation about what `"permission": "read"` exactly means compared to `"field": {"bsn": "read"}`.

Reply:
I honestly don't know what it means.

Reply:
Permission "read" means user can see original value of the field in database. (e.g. default for all). It is used to give access to individual fields using Profile.